specification/purview/data-plane/Azure.Analytics.Purview.Insights/preview/2022-03-01-preview/purviewinsights.json

```diff
@@ -0,0 +1,1363 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "MicrosoftAzurePurviewInsightsClient",
+    "description": "Creates a Microsoft.Azure.Purview.Insights management client.",
+    "version": "2022-03-01-preview"
+  },
+  "host": "purview.azure.com",
+  "basePath": "/insights",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/assets/snapshots": {
+      "get": {
+        "tags": [
+          "Assets"
+        ],
+        "summary": "List asset snapshots",
+        "description": "List asset snapshots",
+        "operationId": "AssetSnapshots_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "kind",
+            "description": "The asset type.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "ActiveUsersTrend",
+              "AssetFilesAggregation",
+              "AssetDistribution",
+              "DataEstateHealth",
+              "FilesWithoutResourceSet",
+              "AllAssetsDistribution",
+              "AssetCurationDistribution",
+              "DeletedAssetsDistribution",
+              "MostViewedAssets",
+              "NetNewAssetsDistribution",
+              "TopSearchedKeywords",
+              "Tags",
+              "TrendsAndGaps"
+            ],
+            "x-ms-enum": {
+              "name": "AssetType",
+              "modelAsString": true
+            }
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Snapshot"
+            }
+          },
```

Comment:
First, this should be [pageable](https://github.com/stankovski/AutoRest/blob/master/Documentation/swagger-extensions.md#x-ms-pageable)

And the type name of the response seems not very appropriate. Usually it would be Snapshots or SnapshotListResult.

Please check other places as well.

Reply:
I will sync it with Details and make it plural as Snapshots